### ISSUE TITLE
Fake code released: Don't share wrong codes which will guide falsely

### ISSUE THREAD

**Issue by ChidanandKumarKS** [RodrigoGantier/SPADE_E2VID#7]
@RodrigoGantier 
I believe the code you have shared for training is not exactly reported in the paper. It seems you have hidden the codes deliberately or you are unable to reach results. 
Next time please don't share wrong codes

**Comment by i-am-the-light** [comment]
> @RodrigoGantier I believe the code you have shared for training is not exactly reported in the paper. It seems you have hidden the codes deliberately or you are unable to reach results. Next time please don't share wrong codes

yes, I truly agree with you!!!!  many place to rectify ! I repair it for 5 days and doesn't run succesfully until now! Bro, do you fix it out?

**Comment by chowkamlee81** [comment]
@i-am-the-light 
Bro use https://github.com/TimoStoff/event_cnn_minimal to get the data preparation for training data. In this framework, use SPADE-E2VId architecture. Author has used more data to get results using e2vid simulator which he hasn't oiutsourced. Kindly get the data and use spade-e2vid architecture to get better results. Real problem is in data
